syntax = "proto3";

package point_pb2;

service PointService {
  rpc GetPointStream (Empty) returns (stream Point);
}

service BluetoothService {
  rpc Connect (BluetoothRequest) returns (BluetoothResponse);
}

message Empty {}

message Point {
  float x = 1;
  float y = 2;
  float z = 3;
  float roll = 4;
  float pitch = 5;
  float yaw = 6;
}

message BluetoothRequest {
  string uart_string = 1;
}

message BluetoothResponse {
  string status = 1;
}
